indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,60 +1,112 @@
 {% from 'message_box.html' import message_box %}
 
-{% macro render_notification_list(event) -%}
+{% macro render_notification_list(event, tracks, states, contrib_types, email_tpls) -%}
     {% if not event.abstract_email_templates %}
         {% call message_box('info', classes='no-rules-placeholder') %}
             {% trans %}There are currently no e-mail templates{% endtrans%}
         {% endcall %}
     {% else %}
-        <ul class="email-template-list" data-url="{{ url_for('.email_tpl_sort', event) }}">
-            {% for tpl in event.abstract_email_templates|sort(attribute='position') %}
-                <li class="email-template i-box" data-id="{{ tpl.id }}">
-                    <div class="ui-i-box-sortable-handle"></div>
-                    <div class="i-box-header">
-                        <span class="i-box-title">{{ tpl.title }}</span>
-                        <div class="i-box-buttons toolbar right">
-                            <div class="group">
-                                <a class="i-button icon-eye"
-                                   data-href="{{ url_for('.email_tpl_preview', tpl) }}"
-                                   data-method="GET"
-                                   data-title="{% trans %}E-mail preview{% endtrans %}"
-                                   data-ajax-dialog>
-                                   {% trans %}Preview{% endtrans %}
-                                </a>
-                                <a class="i-button icon-edit arrow js-edit-tpl-dropdown"
-                                   data-toggle="dropdown">{% trans %}Edit{% endtrans %}</a>
-                                <ul class="dropdown">
-                                    <li>
-                                        <a data-href="{{ url_for('.email_tpl_rule_edit', tpl) }}"
-                                           title="{% trans %}Edit ruleset{% endtrans %}"
-                                           data-title="{% trans %}Edit ruleset{% endtrans %}"
-                                           data-ajax-dialog>
-                                           {% trans %}Ruleset{% endtrans %}
+        <div class="email-template-container">
```

Comment:
The semantic of the classes can be improved. Here you have `email-template-container` which suggests that inside I'll find an `email-template`. However, I find the list of email templates. Also, the `email-template-notifications` doesn't make much sense, the feature is Email Notification Templates (or Email Templates or Notification Templates). Bringing `template` before the `notifications` is a bit confusing.
I would suggest the following structure:
```
div.email-templates-wrapper
    div.email-templates
        ul
            li.email-template
```
instead of 
```
div.email-template-container
    div.email-template-notifications
        ul.email-template-list
            li.email-template
```
The `ul` can easily be selected with `$('.email-templates ul`), no need to add a class imo